gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -223,6 +223,8 @@ domains:
 #    algorithm: None
 #    properties:
 #      rounds: 10
+# Define max number of secrets for application.
+#  secretsMax: 10
```

Comment:
todo: also need to be applied on the Gateway config file to be used by DCR request